core/templates/pages/skill-editor-page/modal-templates/add-worked-example-modal.directive.html

```diff
@@ -6,15 +6,25 @@ <h3>Add Worked Example</h3>
   <form name="addWorkedExampleForm.workedExampleTextForm"
         class="form-inline">
     <div class="oppia-rule-details-header">
-      <strong>Worked Example</strong>
+      <strong>Example Question</strong>
       <schema-based-editor schema="WORKED_EXAMPLE_FORM_SCHEMA"
-                           local-value="tmpWorkedExampleHtml">
+                           local-value="tmpWorkedExampleQuestionHtml">
       </schema-based-editor>
+      <div style="margin-top: 2.5vh;">
```

Comment:
Can we separate this out into a class name like in `worked-example-editor.directive.html`.

Reply:
Done